astroquery/sdss/core.py

```diff
@@ -520,7 +520,7 @@ class = 'galaxy' \
 
     def get_spectra_async(self, coordinates=None, radius=2. * u.arcsec,
                           matches=None, plate=None, fiberID=None, mjd=None,
-                          timeout=TIMEOUT,
+                          timeout=TIMEOUT, get_query_payload=False,
```

Comment:
the order of the kwargs are different than in the previous release (before the previous PR). So while it's not strictly related, making all optional arguments keyword only is the way to go here.